src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background-color: $white-color;
+  width: 200px;
+  padding: 32px 16px 16px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), values in shorthand properties should be ordered consistently. The padding shorthand should follow the order: top, right, bottom, left. If top and bottom paddings are the same, and left and right paddings are the same, you can use the shorthand with two values.